nursery/encrypt-data-using-curve25519.yml

```
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: encrypt data using Curve25519
+    namespace: data-manipulation/encryption/curve25519
+    author: dimiter.andonov@fireeye.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    examples:
+      - 0a0882b8da225406cc838991b5f67d11:0x4135f6, 0x416f51
```

one example per line, please, the test harness relies on this format
```suggestion
      - 0a0882b8da225406cc838991b5f67d11:0x4135f6
      - 0a0882b8da225406cc838991b5f67d11:0x416f51
```